source/whats-new.txt

```diff
@@ -12,22 +12,71 @@ What's New
 
 Learn what's new in:
 
+* :ref:`Version 4.3 <version-4.3>`
 * :ref:`Version 4.2 <version-4.2>`
 * :ref:`Version 4.1 <version-4.1>`
 * :ref:`Version 4.0 <version-4.0>`
 
+.. _version-4.3:
+
+What's New in 4.3
+-----------------
+
+New features of the 4.3 Java driver release include:
+
+- MongoDB Versioned API. For more information, see our :doc:`Versioned API guide </fundamentals/versioned-api>`. 
+- Added a ``setWindowFields`` pipeline stage to allow the use of window operators 
```

Comment:
Change

    Added a ``setWindowFields`` pipeline stage

to

    Added a builder API for the ``setWindowFields`` pipeline stage